app/javascript/components/App.js

```diff
@@ -64,21 +62,17 @@ export class App extends Component {
       method: "PATCH",
     })
       .then((response) => response.json())
-      .then((payload) => this.readBlog())
+      .then(() => this.readBlog())
       .catch((errors) => console.log("Blog update errors: ", errors));
   };
 
   deleteBlog = (id) => {
     fetch(`http://localhost:3000/blogs/${id}`, {
       headers: {
-        "Content-Type": "application/json"
+        "Content-Type": "application/json",
       },
-      method: "DELETE"
-    })
-    .then(response => response.json()
-    .this(payload => this.readBlog())
-    
-    )
+      method: "DELETE",
+    }).then((response) => response.json().then((payload) => this.readBlog()));
```

Comment:
Let's keep method chains consistent. In other places, you start a new line for each `.then`, so consider doing that here.